### PR TITLE
update FAQ: portal doesn't list all repositories/tags

### PR DESCRIPTION
**Pull request by ankurkhemani**

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -28,6 +28,7 @@
     - [How to enable and get the debug logs of docker daemon?](#how-to-enable-and-get-the-debug-logs-of-docker-daemon)
     - [New user permissions may not be effective immediately after updating](#new-user-permissions-may-not-be-effective-immediately-after-updating)
     - [Authentication information is not given in the correct format on direct REST API calls](#authentication-information-is-not-given-in-the-correct-format-on-direct-rest-api-calls)
+    - [Portal doesn't list all repositories/tags](#Portal-doesn't-list-all-repositories/tags)
```
Comment (sajayantony) [inline docs/FAQ.md]: Recommend rephrasing this to "Why does the Azure Portal not list all my repositories?"

**Comment by sajayantony** [comment]
@dlepow - could you help review this or should we send a PR over your current PR?

File: docs/FAQ.md (hunk 1 above)
Comment (ankurkhemani) [reply docs/FAQ.md]: Rephrased.

**Comment by dlepow** [comment]
@ankurkhemani @sajayantony - I will add this info to the FAQ which is publishing to docs within the next day or so. I'll also submit a PR soon to deprecate the FAQ in this acr repo.

**Pull request merged**